packages/babel-plugin-transform-runtime/scripts/build-dist.js

```diff
@@ -0,0 +1,158 @@
+"use strict";
+
+const path = require("path");
+const outputFile = require("output-file-sync");
+const helpers = require("@babel/helpers");
+const babel = require("@babel/core");
+const template = require("@babel/template");
+const t = require("@babel/types");
+
+const transformRuntime = require("../");
+const corejs2Definitions = require("../lib/definitions").default;
+
+writeHelpers("@babel/runtime");
+
+writeCoreJS2("@babel/runtime-corejs2");
+writeHelpers("@babel/runtime-corejs2", { corejsVersion: 2 });
+
+function writeCoreJS2(runtimeName) {
+  const pkgDirname = getRuntimeRoot(runtimeName);
+
+  const paths = ["is-iterable", "get-iterator"];
+
+  Object.keys(corejs2Definitions.builtins).forEach(key => {
+    const path = corejs2Definitions.builtins[key];
+    paths.push(path);
+  });
+
+  Object.keys(corejs2Definitions.methods).forEach(key => {
+    const props = corejs2Definitions.methods[key];
+    Object.keys(props).forEach(key2 => {
+      paths.push(props[key2]);
+    });
+  });
+
+  paths.forEach(function(corePath) {
+    outputFile(
+      path.join(pkgDirname, "core-js", `${corePath}.js`),
+      `module.exports = require("core-js/library/fn/${corePath}");`
+    );
+  });
+}
+
+function writeHelpers(runtimeName, { corejsVersion } = {}) {
+  writeHelperFiles(runtimeName, { corejsVersion, esm: false });
+  writeHelperFiles(runtimeName, { corejsVersion, esm: true });
+}
+
+function writeHelperFiles(runtimeName, { esm, corejsVersion }) {
+  const pkgDirname = getRuntimeRoot(runtimeName);
+
+  for (const helperName of helpers.list) {
+    const helperFilename = path.join(
+      pkgDirname,
+      "helpers",
+      esm ? "es6" : "",
+      `${helperName}.js`
+    );
+
+    outputFile(
+      helperFilename,
+      buildHelper(runtimeName, pkgDirname, helperFilename, helperName, {
+        esm,
+        corejsVersion,
+      })
+    );
+  }
+}
+
+function getRuntimeRoot(runtimeName) {
+  return path.resolve(
+    __dirname,
+    "..",
+    "..",
+    runtimeName.replace(/^@babel\//, "babel-")
+  );
+}
+
+function buildHelper(
+  runtimeName,
+  pkgDirname,
+  helperFilename,
+  helperName,
+  { esm, corejsVersion }
+) {
+  const tree = t.program([], [], esm ? "module" : "script");
+  const dependencies = {};
+  let bindings = null;
+
+  if (!esm) {
+    bindings = [];
+    for (const dep of helpers.getDependencies(helperName)) {
+      const id = (dependencies[dep] = t.identifier(t.toIdentifier(dep)));
+      tree.body.push(template.statement.ast`
+        var ${id} = require("${`./${dep}`}");
```

Comment:
Some helpers redefine the exported function. Does this work in that case?

Reply:
Yeah, it's a bug, but it's existing and tracked in https://github.com/babel/babel/issues/8087

Reply:
Oh ok, I didn't realize you just moved around this code